packages/site/src/config/theme.ts

```diff
@@ -0,0 +1,181 @@
+import { createGlobalStyle, DefaultTheme } from 'styled-components';
+
+const breakpoints = ['600px', '768px', '992px'];
+
+/**
+ * Common theme properties.
+ */
+const theme = {
+  fonts: {
+    default:
+      '-apple-system,BlinkMacSystemFont,"Segoe UI",Roboto,Oxygen-Sans,Ubuntu,Cantarell,"Helvetica Neue",sans-serif',
+    code: 'ui-monospace,Menlo,Monaco,"Cascadia Mono","Segoe UI Mono","Roboto Mono","Oxygen Mono","Ubuntu Monospace","Source Code Pro","Fira Mono","Droid Sans Mono","Courier New", monospace',
+  },
+  fontSizes: {
+    heading: '52px',
+    mobileHeading: '36px',
+    title: '24px',
+    large: '20px',
+    default: '16px',
+    small: '14px',
+  },
```

Comment:
Can we use rem for good practice?

Reply:
I used pixels to have something simple to understand even if you're not a front-end person but looking at the overall complexity of the template I agree we could use `rem` for good practices

Reply:
#2